yarn.lock

```diff
@@ -11536,11 +11536,16 @@ typedarray@^0.0.6:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-5.3.3.tgz#b3ce6ba258e72e6305ba66f5c9b452aaee3ffe37"
   integrity sha512-pXWcraxM0uxAS+tN0AG/BF2TyqmHO014Z070UsJ+pFvYuRSq8KH8DmWpnbXe0pEPDHXZV3FcAbJkijJ5oNEnWw==
 
-typescript@^4.8.3, typescript@^4.9.5:
+typescript@^4.9.5:
```

Comment:
derp we should probably upgrade `eslint-plugin` as well, that seems to be where this comes from (use `yarn why typescript` to confirm)

Reply:
@ltrouton is upgrading Typescript in eslint-plugin.